langchain/llms/huggingface_pipeline.py

```diff
@@ -115,7 +115,10 @@ def from_model_id(
                     "can be a positive integer associated with CUDA device id.",
                     cuda_device_count,
                 )
-
+        if "trust_remote_code" in _model_kwargs:
+            _model_kwargs = {
+                k: v for k, v in _model_kwargs.items() if k != "trust_remote_code"
```

Comment:
could we just del _model_kwargs["trust_remote_code"] or _model_kwargs.pop("trust_remote_code")

Reply:
what's wrong with mutating input if you're resetting the var anyways? are you worried someone is using this dict outside this function?

Reply:
Yeah there's a chance someone would want to instantiate two different LLMs with the same model kwargs for instance